matrix_sdk/src/request_builder.rs

```diff
@@ -222,38 +229,30 @@ pub struct MessagesRequestBuilder {
     /// The maximum number of events to return.
     ///
     /// Default: 10.
-    limit: Option<UInt>,
+    limit: Option<u32>,
     /// A filter of the returned events with.
     filter: Option<RoomEventFilter>,
 }
 
 impl MessagesRequestBuilder {
     /// Create a `MessagesRequestBuilder` builder to make a `get_message_events::Request`.
-    ///
     /// The `room_id` and `from`` fields **need to be set** to create the request.
```

Comment:
We should change this doc comment now that it's enforced by our type system and document the arguments in their doc section.